src/main/java/de/tum/cit/aet/artemis/lti/service/LtiDeepLinkingService.java

```diff
@@ -116,40 +116,40 @@ private String buildLtiDeepLinkResponse(String clientRegistrationId, Lti13DeepLi
     /**
      * Maps each exercise ID to an individual LTI content item.
      */
-    private List<LtiContentItem> populateExerciseContentItems(String courseId, Set<Long> exerciseIds) {
+    private List<LtiContentItem> populateExerciseContentItems(Long courseId, Set<Long> exerciseIds) {
```

Comment:
Why are all the `Long`s boxed? I'd appreciate if they're primitive. All the methods would work anyways only properly with an existing value for it. The possibility to call it with a null value makes no sense.